azure-ts-appservice-devops/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-devops/infra)
 
-# A Todo App on Azure App Service with SQL Database and Application Insights and deploys it to Azure DevOps
+# Todo App with SQL Database and Application Insights Integrated with Azure DevOps
```

Comment:
App Service is probably more relevant than e.g. Application Insights to stay in the title